messages/main.go

```diff
@@ -55,7 +57,8 @@ func Handler(request Request) (Response, error) {
 	}
 	log.Printf("Processing an event of outer type %s", eventsAPIEvent.Type)
 
-	handleSlackChallenge(eventsAPIEvent, body, resp)
+	handleSlackChallenge(eventsAPIEvent, body, &resp)
+	analyzeAndStore(eventsAPIEvent, &resp)
```

Comment:
This should only be ran when a user writes a message, so under `if eventsAPIEvent.Type == slackevents.CallbackEvent` (and maybe even an extra case based on innverEvent.Data.(type)

Otherwise we risk collecting data that are not actual users writing their messages, and polluting our "messages" DB with slack events that are not messages

Reply:
ALso as we said, split this in 3 :
1. First Store the incoming slack message
2. Run the NLP
3. Update the DynaomDB Item with NLP data